Method,Name,Error,Occurrences
GET,/Categorias,"ConnectionResetError(10054, 'Se ha forzado la interrupción de una conexión existente por el host remoto', None, 10054, None)",9
PUT,Abrir Mesa,CatchResponseError('Fallo al abrir mesa 7'),48
PUT,Cerrar Mesa,CatchResponseError('Fallo al cerrar mesa 6'),136
GET,/Productos,"ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x....>, 'Connection to 192.168.100.15 timed out. (connect timeout=None)')",9
POST,Agregar Item,CatchResponseError('Fallo al agregar item 1 en mesa 3'),94
PUT,Cerrar Mesa,CatchResponseError('Fallo al cerrar mesa 8'),103
GET,/Categorias,"ConnectionAbortedError(10053, 'Se ha anulado una conexión establecida por el software en su equipo host', None, 10053, None)",19
POST,Agregar Item,CatchResponseError('Fallo al agregar item 3 en mesa 2'),108
POST,Agregar Item,CatchResponseError('Fallo al agregar item 3 en mesa 3'),99
GET,/Categorias,"ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x....>, 'Connection to 192.168.100.15 timed out. (connect timeout=None)')",63
POST,Agregar Item,CatchResponseError('Fallo al agregar item 1 en mesa 4'),118
POST,Agregar Item,CatchResponseError('Fallo al agregar item 4 en mesa 7'),105
PUT,Abrir Mesa,CatchResponseError('Fallo al abrir mesa 1'),59
POST,Agregar Item,CatchResponseError('Fallo al agregar item 4 en mesa 4'),106
GET,/Pedidos/5925,HTTPError('500 Server Error: Internal Server Error for url: /Pedidos/5925'),3
POST,Agregar Item,CatchResponseError('Fallo al agregar item 4 en mesa 12'),85
PUT,Abrir Mesa,CatchResponseError('Fallo al abrir mesa 11'),51
GET,/Mesas,RemoteDisconnected('Remote end closed connection without response'),1022
POST,Agregar Item,CatchResponseError('Fallo al agregar item 2 en mesa 9'),112
PUT,Abrir Mesa,CatchResponseError('Fallo al abrir mesa 3'),60
POST,Agregar Item,CatchResponseError('Fallo al agregar item 3 en mesa 7'),98
PUT,Cerrar Mesa,CatchResponseError('Fallo al cerrar mesa 9'),121
GET,/Productos,"ConnectionResetError(10054, 'Se ha forzado la interrupción de una conexión existente por el host remoto', None, 10054, None)",3
POST,Agregar Item,CatchResponseError('Fallo al agregar item 3 en mesa 9'),124
GET,/Pedidos/5925,RemoteDisconnected('Remote end closed connection without response'),3010
POST,Agregar Item,CatchResponseError('Fallo al agregar item 2 en mesa 12'),131
PUT,Cerrar Mesa,CatchResponseError('Fallo al cerrar mesa 2'),141
GET,/Mesas,"ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x....>, 'Connection to 192.168.100.15 timed out. (connect timeout=None)')",18
POST,Agregar Item,CatchResponseError('Fallo al agregar item 1 en mesa 11'),117
POST,Agregar Item,CatchResponseError('Fallo al agregar item 2 en mesa 3'),113
GET,/Pedidos/5925,"ConnectionResetError(10054, 'Se ha forzado la interrupción de una conexión existente por el host remoto', None, 10054, None)",7
POST,Agregar Item,CatchResponseError('Fallo al agregar item 4 en mesa 9'),115
GET,/Categorias,HTTPError('500 Server Error: Internal Server Error for url: /Categorias'),4
GET,/Mesas,"ConnectionAbortedError(10053, 'Se ha anulado una conexión establecida por el software en su equipo host', None, 10053, None)",6
PUT,Cerrar Mesa,CatchResponseError('Fallo al cerrar mesa 7'),125
POST,Agregar Item,CatchResponseError('Fallo al agregar item 3 en mesa 1'),120
PUT,Cerrar Mesa,CatchResponseError('Fallo al cerrar mesa 10'),155
POST,Agregar Item,CatchResponseError('Fallo al agregar item 3 en mesa 13'),112
GET,/Mesas,HTTPError('500 Server Error: Internal Server Error for url: /Mesas'),1
POST,Agregar Item,CatchResponseError('Fallo al agregar item 1 en mesa 8'),143
GET,/Pedidos/5925,"ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x....>, 'Connection to 192.168.100.15 timed out. (connect timeout=None)')",44
POST,Agregar Item,CatchResponseError('Fallo al agregar item 1 en mesa 1'),119
POST,Agregar Item,CatchResponseError('Fallo al agregar item 3 en mesa 12'),112
GET,/Categorias,RemoteDisconnected('Remote end closed connection without response'),4017
POST,Agregar Item,CatchResponseError('Fallo al agregar item 4 en mesa 2'),95
POST,Agregar Item,CatchResponseError('Fallo al agregar item 4 en mesa 13'),119
GET,/Mesas,"ConnectionResetError(10054, 'Se ha forzado la interrupción de una conexión existente por el host remoto', None, 10054, None)",3
PUT,Abrir Mesa,CatchResponseError('Fallo al abrir mesa 5'),42
PUT,Abrir Mesa,CatchResponseError('Fallo al abrir mesa 8'),48
POST,Agregar Item,CatchResponseError('Fallo al agregar item 1 en mesa 5'),97
POST,Agregar Item,CatchResponseError('Fallo al agregar item 2 en mesa 5'),103
POST,Agregar Item,CatchResponseError('Fallo al agregar item 4 en mesa 1'),123
POST,Agregar Item,CatchResponseError('Fallo al agregar item 1 en mesa 12'),128
POST,Agregar Item,CatchResponseError('Fallo al agregar item 1 en mesa 9'),111
GET,/Productos,"ConnectionAbortedError(10053, 'Se ha anulado una conexión establecida por el software en su equipo host', None, 10053, None)",7
PUT,Abrir Mesa,CatchResponseError('Fallo al abrir mesa 10'),40
PUT,Cerrar Mesa,CatchResponseError('Fallo al cerrar mesa 1'),145
PUT,Abrir Mesa,CatchResponseError('Fallo al abrir mesa 2'),42
POST,Agregar Item,CatchResponseError('Fallo al agregar item 2 en mesa 8'),104
PUT,Cerrar Mesa,CatchResponseError('Fallo al cerrar mesa 4'),141
POST,Agregar Item,CatchResponseError('Fallo al agregar item 3 en mesa 11'),110
POST,Agregar Item,CatchResponseError('Fallo al agregar item 1 en mesa 2'),108
POST,Agregar Item,CatchResponseError('Fallo al agregar item 1 en mesa 7'),109
PUT,Cerrar Mesa,CatchResponseError('Fallo al cerrar mesa 3'),110
PUT,Cerrar Mesa,CatchResponseError('Fallo al cerrar mesa 5'),138
POST,Agregar Item,CatchResponseError('Fallo al agregar item 2 en mesa 13'),117
PUT,Abrir Mesa,CatchResponseError('Fallo al abrir mesa 6'),37
GET,/Pedidos/5925,"ConnectionAbortedError(10053, 'Se ha anulado una conexión establecida por el software en su equipo host', None, 10053, None)",11
PUT,Abrir Mesa,CatchResponseError('Fallo al abrir mesa 9'),65
PUT,Abrir Mesa,CatchResponseError('Fallo al abrir mesa 4'),50
POST,Agregar Item,CatchResponseError('Fallo al agregar item 2 en mesa 11'),127
GET,/Productos,RemoteDisconnected('Remote end closed connection without response'),613
POST,Agregar Item,CatchResponseError('Fallo al agregar item 4 en mesa 11'),91
POST,Agregar Item,CatchResponseError('Fallo al agregar item 1 en mesa 13'),100
POST,Agregar Item,CatchResponseError('Fallo al agregar item 3 en mesa 4'),119
POST,Agregar Item,CatchResponseError('Fallo al agregar item 2 en mesa 2'),93
POST,Agregar Item,CatchResponseError('Fallo al agregar item 4 en mesa 5'),107
POST,Agregar Item,CatchResponseError('Fallo al agregar item 4 en mesa 8'),96
POST,Agregar Item,CatchResponseError('Fallo al agregar item 2 en mesa 1'),122
POST,Agregar Item,CatchResponseError('Fallo al agregar item 3 en mesa 5'),120
POST,Agregar Item,CatchResponseError('Fallo al agregar item 3 en mesa 8'),117
POST,Agregar Item,CatchResponseError('Fallo al agregar item 2 en mesa 4'),118
PUT,Cerrar Mesa,CatchResponseError('Fallo al cerrar mesa 11'),132
POST,Agregar Item,CatchResponseError('Fallo al agregar item 4 en mesa 3'),122
POST,Agregar Item,CatchResponseError('Fallo al agregar item 2 en mesa 7'),110
